assets/js/type-defs/cart.js

```diff
@@ -92,31 +92,40 @@
- * @property {string} currency_prefix             Price prefix for the currency
- *                                                which can be used to format
- *                                                returned prices.
- * @property {string} currency_suffix             Price suffix for the currency
- *                                                which can be used to format
- *                                                returned prices.
- * @property {string} currency_decimal_separator  The string used for the
- *                                                decimal separator.
- * @property {string} currency_thousand_separator The string used for the
- *                                                thousands separator.
- * @property {string} price                       Current product price.
- * @property {string} regular_price               Regular product price.
- * @property {string} sale_price                  Sale product price, if
- *                                                applicable.
- * @property {Object} price_range                 Price range, if applicable.
- * @property {string} price_range.min_amount      Price min amount in range.
- * @property {string} price_range.max_amount      Price max amount in range.
+ * @property {string}      currency_code               The ISO code for the
+ *                                                     currency.
+ * @property {number}      currency_minor_unit         The precision (decimal
+ *                                                     places).
+ * @property {string}      currency_symbol             The symbol for the
+ *                                                     currency (eg '$')
+ * @property {string}      currency_prefix             Price prefix for the
+ *                                                     currency which can be
+ *                                                     used to format returned
+ *                                                     prices.
+ * @property {string}      currency_suffix             Price suffix for the
+ *                                                     currency which can be
+ *                                                     used to format returned
+ *                                                     prices.
+ * @property {string}      currency_decimal_separator  The string used for the
+ *                                                     decimal separator.
+ * @property {string}      currency_thousand_separator The string used for the
+ *                                                     thousands separator.
+ * @property {string}      price                       Current product price.
+ * @property {string}      regular_price               Regular product price.
+ * @property {string}      sale_price                  Sale product price, if
+ *                                                     applicable.
+ * @property {CartItemPriceRange|null} price_range     Price range, if
```

Comment:
heh is the column width we've been using a standard, or preference?

Reply:
Loosely convention (based on GB).  I chose to break convention here thought because the last column would end up being super small (if I followed hard 80 line rule - which I generally do)